org.lflang/src/org/lflang/generator/GeneratorBase.java

```diff
@@ -296,6 +296,11 @@ public void doGenerate(Resource resource, LFGeneratorContext context) {
             targetConfig,
             errorReporter
         );
+
+        // If fastMode is enabled, report error on the AST Node 
+        if (!ASTUtils.allPhysicalActions(this.reactors).isEmpty() && targetConfig.fastMode) {
+            errorReporter.reportError(targetConfig.fastModeASTNode, "When physical action is present `fast` must be false.");
```

Comment:
Should this really be an error? The user could have included a reactor that contains physical actions internally without knowing. That would break their build, even though `fast` is not supposed to change the semantics of the program but just be an optimization AFAIU. I think a warning would be more appropriate

I also prefer to avoid giving commands in messages to the user ("fast must be false"). I find a warning message like the following feels nicer: "Fast mode cannot be enabled in a program with physical actions, this will be ignored."

Reply:
I’ll make changes accordingly and discuss this with @lhstrh in our next meeting

Reply:
I like @oowekyala's suggestion to ignore the `fast` flag and showing a warning that explains why.

Reply:
If we have consensus I will open an issue potentially changing error here to warning?
https://github.com/lf-lang/lingua-franca/blob/d9604195fb1be5abefd715c8a064b8489dd92500/org.lflang/src/org/lflang/validation/LFValidator.java#L1093-L1097

Reply:
I don't think it's worth opening a new issue for that. It's fine to amend #1380.

Reply:
This is for `LFValidator` tho, they already have such functionality but it's an error not warning.